src/cascadia/TerminalSettingsEditor/Profiles.cpp

```diff
@@ -64,6 +100,24 @@ namespace winrt::Microsoft::Terminal::Settings::Editor::implementation
         {
             biButton.IsChecked(biButton.Tag().as<int32_t>() == biAlignmentVal);
         }
+
+        // Set the text disclaimer for the text box
+        // For an unknown reason, data-binding the text does not compile (using x:Bind)
```

Comment:
every compiler error has a reason.

Reply:
(don't leave comments like this.)

Reply:
Removing the "for an unknown reason..." part. IIRC, the error came up in the XAML code, but I don't have XAML symbols with me so I can't look into it all the way.